recipes/testmsm/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "testmsm" %}
+{% set version = "3.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/testmsm-{{ version }}.tar.gz
+  sha256: d6aea2a8820953a5d42059cd411da0f133e90abb91c2d26893046d709fbbe965
+
+build:
+  entry_points:
+    - msmb = msmbuilder.scripts.msmb:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python 
+    - mdtraj
+    - cython
+    - numpy 1.19
+    - libgomp  # [linux]
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - mdtraj
+    - numpy
+    - scikit-learn <=0.24.0
+    - pandas
+    - fastcluster
+    - jinja2 <3.1
+    - pyyaml
+    - pyhmc
+    - pytables
+    - numpydoc
+    - six
+    - scipy
+    - libgomp  # [linux]
+    
+test:
+  imports:
+    - msmbuilder
```

Comment:
I think we should probably test importing the subpackages, since the base `__init__.py` is empty. So add imports like:

```
- msmbuilder.cluster
- msmbuilder.commands
- msmbuilder.decomposition
...
```